src/components/popover/Popover.tsx

```diff
@@ -183,12 +198,20 @@ export const Popover: React.FC<PopoverProps> = (props) => {
       }
       `;
 
-    if (maxHeight) {
-      await setKeyframe(animationFrame);
+      if (maxHeight) {
+        await setKeyframe(animationFrame);
```

Comment:
Please fix this warning showing `'await' has no effect on the type of this expression.ts(80007)`